Add blog post about Belgrade workshop (#164)
usegalaxy-eu/website: _posts/2019-01-19-rna-seq-workshop-belgrade.md

---
site: freiburg

> title: A successuful Galaxy RNA-seq workshop in Belgrade

Reviewer: wonderful or great?

Reviewer 2: ```suggestion
title: A successful Galaxy RNA-seq workshop in Belgrade
```

location: Belgrade, Serbia
tags: [training]
---

> The Freiburg Galaxy team was teaching last week for 2 days in the beautiful city of Belgrade!

Reviewer 2: ```suggestion
Last week, the Freiburg Galaxy team taught a 2 day course in the beautiful city of Belgrade!
```

replace [past continuous](https://www.ef.com/wwen/english-resources/english-grammar/past-continuous-tense/) with past perfect

<div class="multiple-img">
<a data-flickr-embed="true" data-header="true" data-footer="true"  href="https://www.flickr.com/photos/134305289@N03/32922648008/in/shares-7s06ku/" title="Galaxy RNA-seq workshop @ Belgrad"><img src="https://farm8.staticflickr.com/7876/32922648008_1371ba32d5_z.jpg" width="640" height="427" alt="Galaxy RNA-seq workshop @ Belgrad"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
</div>

> This 2 day workshops was organized and hosted by the [Institute of Nuclear Sciences Vinča](http://bg.ac.rs/en/members/institutes/Vinca.php) in Belgrade. Björn, Helena and Bérénice offered there an introductory course to Galaxy and RNA-seq data analysis to 50 participants. 

Reviewer 2: ```suggestion
This 2 day workshop was organized and hosted by the [Institute of Nuclear Sciences Vinča](http://bg.ac.rs/en/members/institutes/Vinca.php) in Belgrade. Björn, Helena and Bérénice offered an introductory course to Galaxy and RNA-seq data analysis for the 50 participants. 
```

> The participants seem to have appreciated this workshop, given the nice feedback we got.

Reviewer 2: ```suggestion
Based on the nice feedback we received, all of the participants seem to have enjoyed this workshop.
```
